packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -98,9 +98,32 @@ export async function load_node({
 				let response;
 
 				if (/^[a-zA-Z]+:/.test(url)) {
```

Comment:
It's fairly hard to tell what this regex is doing now. Could we change it to `/^https?:/` which I think would be equivalent?

Reply:
`/^https?:/` only covers the majority use case and would break more esoteric use cases such as `chrome-extension:`, `ipfs:`, `data:`, `blob:`, which are all technically valid.
![image](https://user-images.githubusercontent.com/1460917/125180404-e645d100-e223-11eb-8b89-f4e080483afe.png)

Reply:
@antony would suggest the solution to this would be to refactor out a variable or function to document it. What do you think about putting a function at the top of the file like:
```
const hasScheme = url => /^[a-zA-Z]+:/.test(url);
```

Then here we could call:
```
if (hasScheme(url)) {
```

I think that'd make it more readable. I know it was already like this and you're not touching this line, but it was just something I noticed when trying to understand this section of code that would have helped me come up-to-speed more quickly
What do you think about that? Obviously it could be inlined, but I think the extra